### PR TITLE
Final Modifications to Scripts and DomUpdater

### PR DESCRIPTION
**Pull request by JWoltjen**
#### What does this PR do?
This PR finalizes scripts.js and domUpdates.js, and makes some final changes to the DOM. 

#### Where should the reviewer start?
The reviewer should start in scripts. 

#### How should this be manually tested?
with NPM test and TDD via chai framework. 

#### Any background context you want to provide?
N/A

### CODE REVIEW

File: src/domUpdates.js [hunk 1]
```diff
@@ -25,10 +27,10 @@ const domUpdates = {
   },
 
   displayDropDown() {
-    document.getElementById('#user-info-dropdown').classList.toggle('hide')
-    document.getElementById('#dropdown-name').innerText = this.currentUser.name
-    document.getElementById('#dropdown-goal').innerText = `Daily Step Goal: ${this.currentUser.dailyStepGoal}`; 
-    document.getElementById('#dropdown-email').innerText = `User Account: ${this.currentUser.email}`; 
+    document.getElementById('user-info-dropdown').classList.toggle('hide')
+    document.getElementById('dropdown-name').innerText = this.currentUser.name
```
Comment (peytonjo) [inline src/domUpdates.js:31]: "comment"

File: src/domUpdates.js [hunk 2]
```diff
@@ -37,39 +39,56 @@ const domUpdates = {
...
-    this.displayUsersStepGoalComparison(); 
-  }, 
+    this.displayStepFriendCard();
+    this.displayUsersStepGoalComparison();
+  },
   displayStepsInfo() {
-    document.getElementById('#input-steps-minutes').value = ''; 
-    document.getElementById('#input-steps').value = ''; 
-    document.qetElementById('steps-info-miles-walked-today').innerText = 
+    document.getElementById('input-steps-minutes').value = '';
```
Comment (peytonjo) [inline src/domUpdates.js:48]: glad we got these changed to getElementByID

File: src/scripts.js [hunk 3]
```diff
@@ -1,35 +1,37 @@
...
 import fetchData from './fetch';
+import domUpdates from './domUpdates';
+import moment from 'moment';
 import UserRepository from './UserRepository';
-import moment from 'moment'
 
 let currentUser
 let todaysDate
 
+/****************SECTIONS******************/
```
Comment (peytonjo) [inline src/scripts.js:12]: glad we got these moved! It makes the code much cleaner

**Review by peytonjo — APPROVED** [review]
Looks great!!

**Review by demaceo — APPROVED** [review]

**Comment by demaceo** [comment]
Much needed refactoring! Thank you

**Pull request merged**